content/tutorial/tutorial.md

```diff
@@ -1106,11 +1113,11 @@ In the Game component's `render` method, we can add the key as `<li key={move}>`
     });
 ```
 
-**[View the full code at this point](https://codepen.io/gaearon/pen/PmmXRE?editors=0010)**
+**[अभी तक का पूरा कोड देखें](https://codepen.io/gaearon/pen/PmmXRE?editors=0010)**
 
-Clicking any of the list item's buttons throws an error because the `jumpTo` method is undefined. Before we implement `jumpTo`, we'll add `stepNumber` to the Game component's state to indicate which step we're currently viewing.
+किसी भी सूची आइटम के बटन पर क्लिक करने से त्रुटि हो जाती है क्योंकि `jumpTo` मेथड अपरिभाषित है। इससे पहले कि हम `jumpTo` को लागू करते हैं, हम गेम घटक की स्थिति में `stepNumber` जोड़ेंगे, यह इंगित करने के लिए कि हम वर्तमान में कौन सा कदम देख रहे हैं।
```

Comment:
हम गेम घटक की स्थिति में `stepNumber` जोड़ेंगे, यह इंगित करने के लिए कि हम वर्तमान में कौन सा कदम देख रहे हैं। -> could be simplified

Reply:
```
हम अभी कोनसे स्टेप पर है, ये दिखाने के लिए हम Game कॉम्पोनेन्ट के स्टेट में `stepNumber` डालेंगे।
```